Libraries/Utilities/Platform.ios.js

```diff
@@ -14,6 +14,7 @@
 
 var Platform = {
   OS: 'ios',
+  select: (obj: Object) => obj.ios,
```

Comment:
Need to make sure that ios is optional. then I have no idea how T would look like in the result value.